harmonica/equivalent_layer/utils.py

```diff
@@ -12,7 +12,9 @@
 
 
 @jit(nopython=True, parallel=True)
-def jacobian_numba(coordinates, points, jac, greens_function):
+def jacobian_numba(
+    coordinates, points, jac, greens_function
+):  # pylint: disable=not-an-iterable
```

Comment:
This is not related to the optimization of forward models, and it will conflict with the changes introduced on #207
```suggestion
```